### PR TITLE
supervision page

### PR DESCRIPTION
**Pull request by MaxDiz**
- New page to capture supervisor capabilities
- Removed supervisor information from Targets page and added to new page
- @isaacholeman has reviewed language

### CODE REVIEW

File: content/en/apps/features/supervision/supervision.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+description: >
+  Supervision and workforce management to strengthen health systems
+keywords: supervisor
+relatedContent: >
+  apps/features/targets
+  design/best-practices/#targets
+  apps/features/integrations/dhis2
+---
+
+Supervision and workforce management are important aspects to building and maintaining high performing community health systems. Supervisors help Community Health Workers to deliver quality healthcare services to their patients through: 1) building CHW care delivery knowledge and skills; 2) fostering a supportive work environment; and 3) supporting continuity of care between home-based care and community health centers or facilities.
```
Comment (andy5995) [inline content/en/apps/features/supervision/supervision.md:13]:
```suggestion
Supervision and workforce management are important aspects to building and maintaining high-performing community health systems. Supervisors help Community Health Workers deliver quality healthcare services to their patients through: 1) building CHW care delivery knowledge and skills; 2) fostering a supportive work environment; and 3) supporting continuity of care between home-based care and community health centers or facilities. 
```

File: content/en/apps/features/supervision/supervision.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  apps/features/integrations/dhis2
+---
+
+Supervision and workforce management are important aspects to building and maintaining high performing community health systems. Supervisors help Community Health Workers to deliver quality healthcare services to their patients through: 1) building CHW care delivery knowledge and skills; 2) fostering a supportive work environment; and 3) supporting continuity of care between home-based care and community health centers or facilities. 
+
+The WHO’s [guidelines](https://www.who.int/hrh/community/guideline-health-support-optimize-hw-programmes/en/) on health policy and system support to optimize community-based health worker programmes emphasizes the importance of regular and systematic CHW supervision. Following these guidelines, the CHT is designed to enable Supervisors to provide personalized performance feedback during CHW supervision as well as track aggregate statistics. Data collected from completed [Care Guides]({{< relref "apps/concepts/care-guides" >}}) produce granular information that can be used to provide coaching to promote compliance with health program standards of practice and monitor outcomes.
```
Comment (andy5995) [inline content/en/apps/features/supervision/supervision.md:15]:
```suggestion
The WHO’s [guidelines](https://www.who.int/hrh/community/guideline-health-support-optimize-hw-programmes/en/) on health policy and system support formulated to optimize community-based health worker programmes emphasizes the importance of regular and systematic CHW supervision. Following these guidelines, the CHT is designed to enable Supervisors to provide personalized performance feedback during CHW supervision as well as track aggregate statistics. Data collected from completed [Care Guides]({{< relref "apps/concepts/care-guides" >}}) produce granular information that can be used to provide coaching that promotes compliance with health program standards of practice and closer monitoring of outcomes.
```

As a layperson, I had trouble with the phrase "health policy and system support to optimize community-based". I verified the intended meaning by looking at the linked article, but thought adding "formulated" helps a bit with the sentence flow (so "system support" appears more clearly as a separate noun)

The other changes may also be due to me being a layperson ;)

File: content/en/apps/features/supervision/supervision.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## CHW Aggregate Targets
+
+{{< figure src="aggregate-supervisor.png" link="aggregate-supervisor.png" class="right col-7 col-lg-6" >}}
+
+For CHW Supervisors, the [Targets]({{< relref "apps/features/targets" >}}) tab provides important insights into how their community unit is progressing towards their goals. It presents Supervisors with actionable information about their CHWs, by aggregating data for each of the CHWs that a Supervisor manages and presenting it in an easily digestible format. This enables Supervisors to gain insight into how well their team of CHWs is working together to meet common goals.
```
Comment (abbyad) [inline content/en/apps/features/supervision/supervision.md:21]: Targets can give a more general view, and not explicitly about goals. Removed it from the first sentence, as we still get into how goals can be used further in the paragraph.

```suggestion
For CHW Supervisors, the [Targets]({{< relref "apps/features/targets" >}}) tab provides important insights into their community unit. It presents Supervisors with actionable information about their CHWs, by aggregating data for each of the CHWs that a Supervisor manages and presenting it in an easily digestible format. This enables Supervisors to gain insight into how well their team of CHWs is working together to meet common goals.
```

File: content/en/apps/features/supervision/supervision.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+Clicking on an aggregate widget opens the detailed view with the data for each individual CHW. If a CHW is performing below the target goal, their value will be highlighted in red, making it easier for Supervisors to know with which CHWs to follow up for coaching and performance management.
```
Comment (abbyad) [inline content/en/apps/features/supervision/supervision.md:23]: We should avoid "clicking" as it implies using a mouse. For touch devices "tap" works well, and "select", "push" are "press" a little more generic so I would use one of those here.

```suggestion
Selecting an aggregate widget opens the detailed view with the data for each individual CHW. If a CHW is performing below the target goal, their value will be highlighted in red, making it easier for Supervisors to know with which CHWs to follow up for coaching and performance management.
```

File: content/en/apps/features/supervision/supervision.md (hunk 2 above)
Comment (abbyad) [inline content/en/apps/features/supervision/supervision.md:15]: We should be using the American variant here: `programs` (careful to leave it as `programmes` in the link!)

File: content/en/apps/features/supervision/supervision.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+## Supervisor Dashboards
+
+{{< figure src="supervisor-dashboards.png" link="supervisor-dashboards.png" class="right col-7 col-lg-6" >}}
+
+Program dashboards track, visualize, and share health progress with stakeholders more broadly. Supervisors can use program dashboards to help articulate their CHW cohorts activities and how they align with program impact standards and indicators. Summary statistics of CHW service area performance (e.g. number of home visits, number of protocol errors, etc) help to identify areas for continued improvement and deeper audits of care data. 
+
+The data that can be visualized is highly configurable, and depends on what data fields are configured in a particular CHT app’s forms. Dashboards can be built using any software that supports visualizing data in the widely used PostgreSQL database. Examples include the open source tool [Superset](https://superset.incubator.apache.org), as well as proprietary technologies like [Klipfolio](https://www.klipfolio.com) and [Tableau](https://www.tableau.com).
```
Comment (abbyad) [inline content/en/apps/features/supervision/supervision.md:33]:
```suggestion
The data that can be visualized is highly configurable, and depends on what data fields are configured in a particular CHT app’s forms. Dashboards can be built using any software that supports visualizing data with the widely used PostgreSQL database. Examples include the open source tool [Superset](https://superset.incubator.apache.org), as well as proprietary technologies like [Klipfolio](https://www.klipfolio.com) and [Tableau](https://www.tableau.com). 
```

File: content/en/apps/features/supervision/supervision.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+{{% pageinfo %}}
+### Deployment Case Study
+[Muso](https://www.musohealth.org), a leading community health organization and major contributor to the CHT, has implemented ”360º supervision”, achieving some of the lowest child mortality rates in sub-Saharan Africa. This model provides dedicated mentorship and supportive supervision to CHWs tailored to each CHW’s particular strengths and challenges. A key theme of our [human-centered approach](https://medicmobile.org/design) was the idea of using data to improve one-to-one supervision, rather than using analytics to replace Supervisors. [Learn more](https://medicmobile.org/blog/new-study-precision-supervision-and-personalized-feedback-dashboards-improve-chw-performance-in-mali)
```
Comment (abbyad) [inline content/en/apps/features/supervision/supervision.md:47]: The "learn more" seems disjointed here, would work better as a full sentence containing a link, or a [read-more link](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#cross-referencing-content).

**Review by abbyad — CHANGES_REQUESTED** [review]
Looks good. Some suggestions inline, then ready to go!

File: content/en/apps/features/supervision/supervision.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MaxDiz) [reply content/en/apps/features/supervision/supervision.md]: conflicted with other suggestion so recreated this change in a new commit

File: content/en/apps/features/supervision/supervision.md (hunk 2 above)
Comment (MaxDiz) [reply content/en/apps/features/supervision/supervision.md]: conflicted with another comment so recreated in a new commit

File: content/en/apps/features/supervision/supervision.md (hunk 6 above)
Comment (MaxDiz) [reply content/en/apps/features/supervision/supervision.md:47]: updated with a full sentence

**Comment by MaxDiz** [comment]
Thanks for the additional feedback @abbyad and @andy5995 . Updated and ready for a final review

**Comment by andy5995** [comment]
> Thanks for the additional feedback @abbyad and @andy5995 . Updated and ready for a final review

You're welcome @MaxDiz :) Looks like you did a lot of work on this!

File: content/en/apps/features/supervision/supervision.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (abbyad) [reply content/en/apps/features/supervision/supervision.md]: I don't see this change in the latest commits.

File: content/en/apps/features/supervision/supervision.md (hunk 2 above)
Comment (abbyad) [reply content/en/apps/features/supervision/supervision.md]: I don't see this updated either.

File: content/en/apps/features/supervision/supervision.md (hunk 6 above)
Comment (abbyad) [reply content/en/apps/features/supervision/supervision.md:47]: I don't see a change here in the [newest version](https://github.com/medic/cht-docs/blob/e2604f4c6a14417701696c44b5e193792bd13d02/content/en/apps/features/supervision/supervision.md).

**Review by abbyad — APPROVED** [review]
It seems as though the some commits you mentioned doing separately were not included. Can you make sure to include those and should be ready to go!

**Pull request merged**